src/utils/automation-helper.js

```diff
@@ -29,10 +29,337 @@ export const getAutomation = async (automationId) => {
+    case 201: {
+      const data = await response.json();
+      const { status, guid, id } = data;
+      const queued = status === 'queued';
+      const runId = typeof id === 'undefined' ? guid : id;
+      if (!triggered || queued) {
+        message = automationRunPolling(automationId, runId, translator);
+      } else {
+        message = parseOutput(data, translator);
+        message.ok = true;
+      }
+      break;
+    }
+    case 400: {
+      message = { message: translator.get('Object.ActionButton.Automation.BadRequest'), ok: false };
+      break;
+    }
+    case 401:
+    case 403: {
+      message = { message: translator.get('Object.ActionButton.Automation.NotAuthorized'), ok: false };
+      break;
+    }
+    case 404: {
+      message = { message: translator.get('Object.ActionButton.Automation.AutomationNotFound'), ok: false };
+      break;
+    }
+    case 500:
+    case 503: {
+      message = { message: translator.get('Object.ActionButton.Automation.AutomationError'), ok: false };
+      break;
+    }
+    default: {
+      message = { message: translator.get('Object.ActionButton.Automation.UnkownError'), ok: false };
+    }
+  }
+  return message;
+};
+
+const applyStyles = (element, styles) => {
+  if (element !== undefined && element !== null) {
+    Object.keys(styles).forEach((key) => {
+      element.style[key] = styles[key];
+    });
+  }
+};
+
+const getSnackbar = (snackbarId) => document.querySelector(`#${snackbarId}`);
+
+const removeSnackbar = (element) => {
+  applyStyles(element, { opacity: 0 });
+  setTimeout(() => {
+    element.remove();
+  }, 1000);
+};
+
+export const createSnackbar = async (message, duration, newTab) => {
+  const randomId = (Math.random() + 1).toString(36).substring(7);
+  const snackbarId = `sn-action-button-snackbar-${randomId}`;
+  const snackContainer = document.createElement('div');
+  snackContainer.setAttribute('id', snackbarId);
+  const transitionMs = 400;
+  const snackContainerStyles = {
+    width: '400px',
+    height: '35px',
+    'background-color': '#FFFFFF',
+    position: 'fixed',
+    left: 'calc(50% - 200px)',
+    right: 'auto',
+    bottom: '24px',
+    'box-shadow': '0px 1px 2px 0px rgb(0 0 0 / 15%)',
+    padding: '6px 16px',
+    'border-radius': '3px',
+    'z-index': 1000,
+    opacity: 0,
+    transition: `visibility 0ms, opacity ${transitionMs}ms linear`,
+  };
+  applyStyles(snackContainer, snackContainerStyles);
+  const contentContainer = document.createElement('div');
+  const contentStyles = {
+    display: 'flex',
+    'justify-content': 'space-between',
+    height: '100%',
+    'align-items': 'center',
+  };
+  applyStyles(contentContainer, contentStyles);
+  const status = document.createElement('span');
+  const iconStyles = {
+    textDecoration: 'none',
+    fontSize: 'inherit',
+  };
+  applyStyles(status, iconStyles);
+  const statusIcon = message.ok ? 'tick' : 'warning';
+  status.setAttribute('class', `lui-icon lui-icon--${statusIcon}`);
```

Comment:
No lui icons outside the client. How will this be handled? Also lui icons are old and we should not use them anymore

Reply:
I added some SVG icons which are the same icons we are using in the hub - open for other better solutions then this as the only other idea I had was bringing in React/Sprout but this may be overkill for only 3 icons :)